lib/storage/WebStorage.ts

```diff
@@ -3,32 +3,28 @@
  * when using LocalStorage APIs in the browser. These events are great because multiple tabs can listen for when
  * data changes and then stay up-to-date with everything happening in Onyx.
  */
-import _ from 'underscore';
 import Storage from './providers/IDBKeyVal';
+import type {KeyList, Key} from './providers/types';
+import type StorageProvider from './providers/types';
 
 const SYNC_ONYX = 'SYNC_ONYX';
 
 /**
  * Raise an event thorough `localStorage` to let other tabs know a value changed
- * @param {String} onyxKey
  */
-function raiseStorageSyncEvent(onyxKey) {
+function raiseStorageSyncEvent(onyxKey: Key) {
     global.localStorage.setItem(SYNC_ONYX, onyxKey);
-    global.localStorage.removeItem(SYNC_ONYX, onyxKey);
+    global.localStorage.removeItem(SYNC_ONYX);
```

Comment:
Why this change?

Reply:
`localStorage.removeItem` only takes one argument